### PR TITLE
SNOW-190226 Document array binding support for the Go connector

### PR DESCRIPTION
**Pull request by sfc-gh-dyoshinaga**
SNOW-190226

### Description
Document array binding support for the Go connector.

### CODE REVIEW

File: doc.go [hunk 1]
```diff
@@ -271,25 +271,34 @@ Footnotes:
...
+	...
+	// Insert the data from the arrays into the table.
+	_, err = db.Exec("insert into my_table values (?, ?, ?, ?)", intArray, fltArray, boolArray, strArray)
+
+Note: For alternative ways to load data into the Snowflake database (including bulk loading using the COPY command), see
+Loading Data Into Snowflake (https://docs.snowflake.com/en/user-guide-data-load.html).
 
-Fetching data isn't an issue because the database data type is provided along
-with the data so the Go Snowflake Driver can translate Snowflake data types to
-Go native data types.
```
Comment (sfc-gh-dyoshinaga) [inline doc.go:288]: Most of this didn't seem to have much to do with binding a time variable to a parameter, so I removed it. There's also an earlier section that discusses the types supported by the driver, so I didn't think there should be another list here.

File: doc.go [hunk 2]
```diff
@@ -271,25 +271,34 @@ Footnotes:
...
+Binding a Parameter to a Time Type
 
-However, when the client binds data to send to the server, the driver cannot
-determine the date/timestamp data types to associate with binding parameters.
-For example:
+Go's database/sql package support the ability to binding a parameter in an SQL statement to a time.Time variable.
```
Comment (sfc-gh-mgilkey) [inline doc.go:299]: Nit: "to binding"?

**Review by sfc-gh-mgilkey — COMMENTED** [review]
LGTM except for one nit.
I'll let Jaewan have the final word.

File: doc.go (hunk 2 above)
Comment (sfc-gh-dyoshinaga) [reply doc.go]: Fixed.

**Review by sfc-gh-jbahk — APPROVED** [review]

**Pull request merged**